### DISCUSSION TITLE
https://5kdk.github.io/blog/2023/11/24/modern-react-deep-dive-review

### DISCUSSION THREAD

**Discussion by giscus** [5kdk/5kdk.github.io#11] [General]
# https://5kdk.github.io/blog/2023/11/24/modern-react-deep-dive-review

모던 리액트 딥다이브를 읽고 남기는 서평입니다.

https://5kdk.github.io/blog/2023/11/24/modern-react-deep-dive-review

**Comment by nocturne9no1** [comment]
좋은 글 잘 봤읍니다.

**Reply by 5kdk** [reply]
감사합니다!
